agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -126,8 +126,8 @@ bool main_thread::init()
             ieee1905_1::eMessageType::AP_AUTOCONFIGURATION_WSC_MESSAGE,
             ieee1905_1::eMessageType::CHANNEL_PREFERENCE_QUERY_MESSAGE,
             ieee1905_1::eMessageType::CHANNEL_SELECTION_REQUEST_MESSAGE,
-            ieee1905_1::eMessageType::ACK_MESSAGE,
-            ieee1905_1::eMessageType::TOPOLOGY_QUERY_MESSAGE})) {
+            ieee1905_1::eMessageType::ACK_MESSAGE, ieee1905_1::eMessageType::TOPOLOGY_QUERY_MESSAGE,
```

Comment:
Why ieee1905_1::eMessageType::TOPOLOGY_QUERY_MESSAGE moved to the same line with ieee1905_1::eMessageType::ACK_MESSAGE? Did the clang move it?

Reply:
agree - @Galvien please add a fixup commit for this.

Reply:
clang moved it because the `))) {` are no longer on the same line, just a comma. Therefore, the TOPOLOGY_QUERY_MESSAGE now fits.